include/tts/tests/range.hpp

```diff
@@ -76,6 +76,9 @@ namespace tts
     static auto prng_seed()       noexcept { return args.seed();     }
     static auto count()           noexcept { return args.count();    }
     static auto ulpmax()          noexcept { return args.ulpmax();   }
+    static auto hex()             noexcept { return args.hex();      }
```

Comment:
you use args.hex() only in the display, no need to access it from a producer, hence this function is unneeded